ios/Classifier/VCPTaxonomy.m

```diff
@@ -133,32 +136,33 @@ - (VCPPrediction *)inflateTopPredictionFromClassification:(MLMultiArray *)classi
 // following
 // https://github.com/inaturalist/inatVisionAPI/blob/multiclass/inferrers/multi_class_inferrer.py#L136
```

Comment:
This link returns a 404 and I can't find anything with that name in the repo. Worth updating or at least removing the reference given that there's a change in the method's algorithm.

Reply:
agreed, removed in ad7287ee40da1879d8d014878a72e6383ec63a7f

we have some documentation of the newer algorithm that the web/api is using, but 1) we're not following that in mobile yet, and 2) it's in a private google doc so I'm not sure if linking it in a public repo is helpful? punting that for later.